.github/workflows/test-and-publish-nightly.yml

```diff
@@ -54,11 +60,14 @@ jobs:
         id: build
         run: |
           set -x
-          case "${{ matrix.runson.name }}" in
+          case "${{ matrix.runson.display }}" in
             macos*)
               brew install gcovr
               make COMMIT_TAG=${GITHUB_SHA:0:7} FAT_BINARY=true release coverage -j
             ;;
+            alpine*)
+              make COMMIT_TAG=$HASH release coverage -j
```

Comment:
Why is `$HASH` for this case and `${GITHUB_SHA:0:7}` for other?

Reply:
`git merge` troubles, fixed in 4618514d160107541e45b9ada181b221eafc4b2f